include/zephyr/bluetooth/audio/pacs.h

```diff
@@ -45,6 +45,31 @@ struct bt_pacs_cap {
 	sys_snode_t _node;
 };
 
+/** Structure for registering PACS */
+struct bt_bap_pacs_register_param {
+	/**
+	 * @brief Enables or disables registration of Sink PAC Characteristic.
+	 */
+	bool snk_pac;
+
+	/**
+	 * @brief Enables or disables registration of Sink Location Characteristic. Registration
+	 * of Sink Location is dependent on snk_pac also being set.
+	 */
```

Comment:
```suggestion
	/**
	 * @brief Enables or disables registration of Sink Location Characteristic. 
	 *
	 * Registration of Sink Location is dependent on @ref bt_bap_pacs_register_param.snk_pac also being set.
	 */
```

Please keep `@brief`s brief :D